examples/websocket.gui_script

```diff
@@ -92,16 +98,33 @@ local function disconnect(self)
 	self.connection = nil
 end
 
+local function listen(self)
```

Comment:
I've managed to lock up the server completely:

* Server: Listen
* Client: Connect
* Client: Send a message
* Server: Stop listening
* Client: Send a message
* Server: Listen <-- NOW IT LOCKS UP THE ENGINE

Reply:
I did this test with the client and server running on the same machine, but I don't that matters.